### ISSUE TITLE
help needed

### ISSUE THREAD

**Issue by pfr-dev** [jcs/sdorfehs-bar#1]
Hey jcs, I'm just wondering if `i3status` is a dependency as I've noticed it referenced in the script. 

Either way, with or without `i3status` installed the script fails. 

Currently I just have a simple shell script pipe to `sdorfehs/bar` but I'd love to get something a little more fancy working but when I go to execute the ruby script from a terminal it kills the terminal without showing any difference to my bar.

I'm obviously not understanding something and I apologise if it's trivial.

```
ruby 3.1.3p185 (2022-11-24 revision 1a6b16756e) [x86_64-netbsd]
```

Cheers

**Comment by jcs** [comment]
Yes, it relies heavily on i3status to probe all the system things and print its output that sdorfehs-bar parses.  This is to avoid forking a bunch of stuff on every update iteration.
